### DISCUSSION TITLE
How do I connect VSCode to existing project with GIT?

### DISCUSSION THREAD

**Discussion by Kodyslaw** [community/community#49785] [New to GitHub]
I'm very new to this, frankly I don't know how to formulate my question properly so I will try to explain my problem as best as I can.

Me and my friends are doing a project of a website. We're using co-pilot and live share extensions when we're all avalible to work togheter. 
But when only one person can work they set up GIT integration of some sorts that lets you push commits straight to the project and makes it so vscode downloads latest version of the website automatically after commits (i think). My problem is I dont know how to connect to that project. I have it downloaded on my PC. I have downloaded and installed GIT as well as extensions friends told me to download for VSCode.

They are way more advanced and i'm trying to catch up, i've been trying to understand how to make cookies in Javascript for few days to add to the website and all of a sudden one of them added a commit that changed fundamentals of the website and frankly I don't even know what's going on anymore. 

Every single tutorial I looked at was about making new project and adding it to your own Github, and not joining already exsiting one.

**Comment by enviGit** [comment]
### Hello Kodyslaw!

- **Obtain the URL of the Git repository:** 
Your friends should give you the URL of the repository they are working on. It should be in the form of _https://github.com/username/repository.git_
- **Clone the repository:** 
Open a terminal or command prompt window, navigate to the folder where you want to clone the repository, and enter the following command: git clone <repository url>. This will download a copy of the repository to your computer.
- **Configure Git:** 
After cloning the repository, you will need to configure Git on your computer so that it knows who you are. Enter the following commands in the terminal window:
```
git config --global user.name "Your Name"
git config --global user.email "youremail@example.com"
```
- **Install dependencies:** 
If the project requires any dependencies to be installed, your friends should provide you with a list of them. Install them using the appropriate package manager (e.g. npm, yarn).
- **Open the project in VS Code:** 
Open VS Code and select "Open Folder". Navigate to the folder where you cloned the repository and select it.
- **Pull changes:** 
Before making any changes to the code, it is important to ensure that you have the latest version of the code. To do this, open the terminal window in VS Code and enter the following command: `git pull`.
- **Make changes and push:** 
After making changes to the code, you can push your changes to the remote repository by entering the following commands in the terminal window:
```
git add .
git commit -m "Your commit message"
git push origin main
```
_Note that "main" should be replaced with the name of the branch that your friends are working on._

**Reply by queenofcorgis** [reply]
Really appreciate the time you took to answer @enviGit - if this helped you @Kodyslaw don't forget to Mark as answer ✅

**Comment by 03timothyash** [comment]
La forma en que ahora van pasando los datos es una otra forma una apps ya nada más estas almacenando tus archivos de una de tus cuentas de correo de Google Gmail. Si pero estas asiendo datos comprimidos no tienen ninguna acción

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: Kodyslaw ***@***.***>
Sent: Saturday, March 11, 2023 5:08:22 AM
To: community/community ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GitHub Community] How do I connect VSCode to existing project with GIT? (Discussion #49785)


I'm very new to this, frankly I don't know how to formulate my question properly so I will try to explain my problem as best as I can.

Me and my friends are doing a project of a website. We're using co-pilot and live share extensions when we're all avalible to work togheter.
But when only one person can work they set up GIT integration of some sorts that lets you push commits straight to the project and makes it so vscode downloads latest version of the website automatically after commits (i think). My problem is I dont know how to connect to that project. I have it downloaded on my PC. I have downloaded and installed GIT as well as extensions friends told me to download for VSCode.

They are way more advanced and i'm trying to catch up, i've been trying to understand how to make cookies in Javascript for few days to add to the website and all of a sudden one of them added a commit that changed fundamentals of the website and frankly I don't even know what's going on anymore.

Every single tutorial I looked at was about making new project and adding it to your own Github, and not joining already exsiting one.

**Comment by 03timothyash** [comment]
Para eso es Android aptoide.

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
